### PR TITLE
[Cookies] Increased cookies banner font size for med/large screens to improve readability

### LINKED ISSUE #2597: [Cookies] Increase font size of cookie banner text on desktop
## Description
When cookie banner display on desktop, font size is pretty low and hard to read, and the page layout is not appealing as text is very low compared to the size of the banner.

## Expected Behavior
Appearance is nice and appealing, and text is easy to read.

## Actual Behavior
It is not

## Steps to Reproduce
1. On an instance with cookie banner enabled, go to homepage
2. See the banner

## Animated Gif/Screenshot
capture du 2018-08-28 08-57-09

## Context
Par of cookie banner epic remaining work.

## Possible Fix
Fix the font size so that it adapts on desktop to the size of the screen and take more space within the banner (as it does already on mobile pretty well).

Shitty done example I did just by resizing the text zone, but you will do it better of course ;-)
capture du 2018-08-28 09-03-57

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

Closes #2597 

Increased font size for med/large screens and adapted CSS to keep the text and button visible in all screen sizes.

#### What should we test?
The cookies banner should be tested in terms of UX in different screen sizes. Functionality was not changed.

#### Release notes
Changelog Category: Changed

Cookies banner font size increased.

### CODE REVIEW

**Comment by luisramos0** [comment]
I am not sure but I dont think the broken build is related to this PR...

the error is:
Using embedded shopfront functionality using iframes allows shopping and checkout
     Failure/Error: expect(page).to have_text 'Your shopping cart'
       expected to find text "Your shopping cart" in "Powered by Open Food Network English Castellano English Login 1 items My Embedded Hub Herndon Next order closing in 2 days Ready for ABOUT MY EMBEDDED HUB PRODUCERS CONTACT GROUPS Ruby on Rails Framed Apples from Enterprise 333 Framed Apples 1g, S $19.99 $19.99". (However, it was found 2 times including non-visible text.)
     # ./spec/features/consumer/shopping/embedded_shopfronts_spec.rb:53:in `block (4 levels) in <top (required)>'

**Comment by mkllnk** [comment]
I just ran Semaphore again and the same spec failed again. I think it needs fixing even though it may be unrelated to this PR.

**Comment by luisramos0** [comment]
I rerun the build and its green now.

**Comment by luisramos0** [comment]
Yes Matt!  I need to leave global config as it was before the test. Which is banner disabled. That is probably causing the error. I'll do that and add a commit to this PR. Also, Ill put that (must keep global state in your tests) to the code conventions.
Thanks!

**Comment by Matt-Yorkley** [comment]
Great! I've seen other random failing feature specs this week for "missing" UI elements which then passed after a restart, so I'm guessing it's the same thing.

**Comment by Matt-Yorkley** [comment]
P.S. @luisramos0 for debugging issues like this locally you can add the line `save_and_open_screenshot` temporarily to a spec at the point where the issue is, and it will give you a screenshot of what capybara is seeing.

**Comment by luisramos0** [comment]
Thanks @Matt-Yorkley 
I didnt know about that trick to get a screenshot, it's great! I added it here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Testing-and-Rspec-Tips#save_and_open_screenshot
I have a little local script that switches to selenium :-)

The convention "leave config as it was before the test" was already here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Testing-and-Rspec-Tips#clearing-app-config-changes-from-cache

I have added a new commit to fix it for the cookies_spec.

Thanks guys!

File: spec/features/consumer/cookies_spec.rb [hunk 1]
```diff
@@ -52,6 +60,14 @@
   end
 
   describe "policy page" do
+
+    # keeps config unchanged
+    around do |example|
+      original_config_value = Spree::Config[:cookies_policy_matomo_section]
```
Comment (Matt-Yorkley) [inline spec/features/consumer/cookies_spec.rb]: I see you've added this for these 2 specs, but is there a global config to disable it on all the other specs by default?

Comment (Matt-Yorkley) [reply spec/features/consumer/cookies_spec.rb]: Ah ok, I wasn't sure about the defaults.

**Comment by luisramos0** [comment]
I just staged this one here: https://staging.openfoodfrance.org/
I also went to the backoffice and activated the banner to have a look myself.

**Comment by myriamboure** [comment]
I used Mozilla "test views" developer tool, pretty useful to check how it looks like on various screen sizes ;-) It doesn't deserve testing note, just a screen share:
It is more readable and decent in term of appearance, so good to go.
capture du 2018-09-03 13-15-45
I checked also on mobile as well of course, all good.

**Pull request merged**